scheduler/group/group.go

```diff
@@ -0,0 +1,61 @@
+package group
+
+import (
+	"runtime"
+
+	"polydawn.net/repeatr/def"
+	"polydawn.net/repeatr/executor"
+	"polydawn.net/repeatr/lib/guid"
+	"polydawn.net/repeatr/scheduler"
+)
+
+// interface assertion
+var _ scheduler.Scheduler = &Scheduler{}
+
+// Dumb struct to send job references back
+type hold struct {
+	forumla  def.Formula
+	response chan def.Job
+}
+
+type Scheduler struct {
+	groupSize int
+	executor  *executor.Executor
+	queue     chan *hold
+}
+
+func (s *Scheduler) Configure(e *executor.Executor) {
+	s.groupSize = runtime.NumCPU()
+	s.executor = e
+	s.queue = make(chan *hold)
+}
+
+func (s *Scheduler) Start() {
+	for w := 1; w <= s.groupSize; w++ {
+		go s.Run()
+	}
+}
+
+func (s *Scheduler) Schedule(f def.Formula) <-chan def.Job {
+
+	h := &hold{
+		forumla:  f,
+		response: make(chan def.Job),
+	}
+
+	go func() {
+		s.queue <- h
+	}()
```

Comment:
Agreed on rube-goldberg status; this is punting pretty hard and will fall apart when things go #webscale, where #webscale is whenever the golang scheduler starts calling my bluff on goroutine madness. From anecdotal personal experience I know that is at least in the tens of thousands, and presumably capable of a few more orders of magnitude. 

Long-term, I'd like both ordering and introspection capability on the queue, only one of which is provided by buffered channels. A steady-state for scheduling interface design is likely to be a long process, so your suggestion sounds fine in the interim.

I'd be comfortable with this PR setting the channels to be buffered, add a Configure param for queue size, and perhaps panicing with a `QueueFullError` if the job cannot be enqueued.